app/controllers/api/v1/npq_applications_controller.rb

```diff
@@ -32,15 +32,11 @@ def reject
       end
 
       def accept
-        profile = npq_lead_provider.npq_profiles.includes(:user, :npq_course).find(params[:id])
-        other_profiles = NPQValidationData.where(profile: (profile.user.npq_profiles - [profile.profile]))
-
-        ActiveRecord::Base.transaction do
-          if profile.update(lead_provider_approval_status: "accepted") && other_profiles.update(lead_provider_approval_status: "rejected")
-            render json: NPQApplicationSerializer.new(profile).serializable_hash
-          else
-            render json: { errors: Api::ErrorFactory.new(model: profile).call }, status: :bad_request
-          end
+        npq_application = npq_lead_provider.npq_applications.includes(:user, :npq_course).find(params[:id])
```

Comment:
I know it was there, but I do wonder why we need the `.includes(:user, :npq_course)`.

Reply:
I don't think we need it, removed